calamus/schema.py

```diff
@@ -128,6 +133,9 @@ def _serialize(self, obj: typing.Union[_T, typing.Iterable[_T]], *, many: bool =
             else:
                 ret[key] = value
 
+        if ("@id" not in ret or not ret["@id"]) and self.opts.id_generation_strategy:
+            ret["@id"] = self.opts.id_generation_strategy(ret, obj)
```

Comment:
Shouldn't we create a default id (using `blank_node_id_strategy`) if `self.opts.id_generation_strategy` is not set?

Reply:
Good idea, thanks